UnityPackageManager/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+	"dependencies": {
```

Comment:
please remove.  I believe this was added to the `.gitignore` in a different PR

Reply:
Good, this file's been driving me nuts.